tools/build_boot_image

```diff
@@ -45,7 +45,7 @@ bi_make_iso=0
 bi_proforma_only=0
 bi_nocleanup=0
 bi_imgsz_gb="2"
```

Comment:
Now this no longer reflects reality, please can you:

 - replace "bi_imgsz_gb" with something like "bi_proforma_prefix=""
 - fix up the -p handling to set that to ${OPTARG}gb and bi_imgsz to $OPTARG * 1000000000

Reply:
Thanks. New commit has replaced variable with new suggested name and fixed up the spots it was used.